configure-mana

```diff
@@ -9,18 +9,21 @@
 MPI_ETHERNET_INTERFACE=\
 `ip addr |grep -B1 link/ether | head -1 |sed -e 's%[^ ]*: \([^ ]*\): .*%\1%'`
 
-# This works on our local test computer (Ubuntu-18.04).
+# Note: modify this path according to your environment.
+MPI_INSTALL_DIR=/usr/local
```

Comment:
Suggest check `MPI_INSTALL_DIR` and only assign default value when it was not set.

BTW, do we use `configure-mana` on CORI?

Reply:
yes we do. i've tested this version of `configure-mana` on Cori and it works.